operator/controllers/reconcilers/server/statefulset_reconciler.go

```diff
@@ -41,14 +41,15 @@ func NewServerStatefulSetReconciler(
 	podSpec *v1.PodSpec,
 	volumeClaimTemplates []mlopsv1alpha1.PersistentVolumeClaim,
 	scaling *mlopsv1alpha1.ScalingSpec,
+	policy *appsv1.StatefulSetPersistentVolumeClaimRetentionPolicy,
```

Comment:
A suggestion to rename this to volumeClaimRetentionPolicy. Mostly because in the future we might end up with more policies for different things, and to eliminate any confusion.

Reply:
Sure, this is update and renamed to `volumeClaimRetentionPolicy`